### PR TITLE
Implementing the stop-propagation helper

### LINKED ISSUE #9: stopPropagation
Pretty much in the same vein as #5.

Relevant Ember Docs: [`{{action}}` Event Propagation](https://www.emberjs.com/api/ember/release/classes/Ember.Templates.helpers/methods/action?anchor=action#event-propagation)

### PR DESCRIPTION
**Pull request by gmurphey**
Closes #9.

### CODE REVIEW

**Comment by gmurphey** [comment]
Thanks for taking a look, @rwjblue! I've added the test for `capture`. Once we have `ember-on-event-helpers` (or whatever it may be called), I'll be happy to move this over there.

File: tests/integration/helpers/stop-propagation-test.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+      </div>
+    `);
+
+    await click('button');
+  });
+
+  test('{{on "click" (stop-propagation) capture=true}}', async function(assert) {
+    assert.expect(1);
+
+    const calls = [];
```
Comment (rwjblue) [inline tests/integration/helpers/stop-propagation-test.js]: Totally not an issue, but I wanted to mention that I've started using `assert.step` / `assert.verifySteps` for these sorts of "order tracking" use cases. This test would look like:

```js
  test('{{on "click" (stop-propagation) capture=true}}', async function(assert) {
    assert.expect(1);

    this.outerListener = () => assert.step('outer');
    this.innerListener = () => assert.step('inner');

     await render(hbs`
      <div {{on "click" (stop-propagation this.outerListener) capture=true}}>
        <button {{on "click" this.innerListener}}>inner</button>
      </div>
    `);

     await click('button');

     assert.verifySteps(['outer'], 'it only runs the outer listener');
  });
```

See:

* https://api.qunitjs.com/assert/step
* https://api.qunitjs.com/assert/verifySteps

**Comment by buschtoens** [comment]
Hey @gmurphey, thanks so much for implementing this! I would love, if you could reopen this PR against [`ember-event-helpers`](https://github.com/buschtoens/ember-event-helpers). ❤️

File: tests/integration/helpers/stop-propagation-test.js (hunk 1 above)
Comment (gmurphey) [reply tests/integration/helpers/stop-propagation-test.js]: Didn’t know about this! Already have ideas how I can use it in other projects. Thanks!

**Comment by gmurphey** [comment]
Closing in favor of buschtoens/ember-event-helpers#4.

**Pull request closed without merging**